### PR TITLE
Update model evaluation and add bootstrap module

### PR DESCRIPTION
**Pull request by nomomon**

### CODE REVIEW

File: text2emoji/models/nn_classifier.py [hunk 1]
```diff
@@ -81,13 +99,13 @@ def get_performance(model, features, target):
...
 
     # Evaluate model
-    predictions = model(features)
-    softmax_predictions = torch.nn.functional.softmax(predictions, dim=1)
-    correct_predictions = softmax_predictions.argmax(dim=1) == target
-    accuracy = correct_predictions.float().mean().item()
+    logits = model(features)
+    probs = torch.nn.functional.softmax(logits, dim=1)
+    preds = probs.argmax(dim=1) == target
+    accuracy = preds.float().mean().item()
```
Comment (Jazhyc) [inline text2emoji/models/nn_classifier.py:105]: Nice, using logits as a variable makes more sense in this scenario

File: text2emoji/models/test_model.py [hunk 2]
```diff
@@ -23,29 +27,83 @@ def test_best_model(type):
...
+        recall_score(eval_labels, random_labels, average="macro"),
+    ]
+    results.loc["model"] = [
+        np.mean(probabilties.argmax(axis=1) == eval_labels),
+        top_k_accuracy_score(eval_labels, probabilties, k=3),
+        top_k_accuracy_score(eval_labels, probabilties, k=5),
+        f1_score(eval_labels, probabilties.argmax(axis=1), average="macro"),
+        precision_score(eval_labels, probabilties.argmax(axis=1), average="macro"),
+        recall_score(eval_labels, probabilties.argmax(axis=1), average="macro"),
+    ]
```
Comment (Jazhyc) [inline text2emoji/models/test_model.py]: I think these statements can be simplified using a loop but it's not too big of a problem

**Review by Jazhyc — APPROVED** [review]
The code looks good. I'm not too familiar with the Bootstrap test but your approach seems logical
